tests/API/Management/UsersTest.php

```diff
@@ -4,61 +4,205 @@
 use Auth0\SDK\API\Management;
 use Auth0\Tests\API\BasicCrudTest;
 
-class UsersTest extends BasicCrudTest {
+/**
+ * Class UsersTest.
+ *
+ * @package Auth0\Tests\API\Management
+ */
+class UsersTest extends BasicCrudTest
+{
+    /**
+     * Unique identifier name for Users.
+     *
+     * @var string
+     */
+    protected $id_name = 'user_id';
 
-    protected $email;
+    /**
+     * Random number used for unique testing names.
+     *
+     * @var integer
+     */
+    protected $rand;
```

Comment:
This seems common across entities tests. What about moving this up to the `BasicCrudTest` class?

Reply:
Good catch ... that was kind of silly :)